app/presenters/task_list/status_tag.rb

```diff
@@ -1,32 +1,43 @@
 module TaskList
-  class StatusTag < BaseRenderer
+  class StatusTag
     attr_reader :status
 
-    DEFAULT_CLASSES = %w[govuk-tag app-task-list__tag].freeze
-
     STATUSES = {
       TaskStatus::COMPLETED => nil,
-      TaskStatus::IN_PROGRESS => 'govuk-tag--blue',
-      TaskStatus::NOT_STARTED => 'govuk-tag--grey',
-      TaskStatus::UNREACHABLE => 'govuk-tag--grey',
-      TaskStatus::NOT_APPLICABLE => 'govuk-tag--grey',
+      TaskStatus::IN_PROGRESS => 'light-blue',
+      TaskStatus::NOT_STARTED => 'blue',
+      TaskStatus::UNREACHABLE => nil,
+      TaskStatus::NOT_APPLICABLE => nil
     }.freeze
 
-    def initialize(crime_application, name:, status:)
-      super(crime_application, name:)
+    def initialize(status:)
       @status = status
     end
 
-    def render
-      tag.strong id: tag_id, class: tag_classes do
-        t!("tasklist.status.#{status}")
-      end
+    def to_hash
```

Comment:
Instead of a render, this constructs arguments that can be used by
https://github.com/x-govuk/govuk-components/blob/main/app/components/govuk_component/task_list_component/status_component.rb